scripts/openroad/replace.tcl

```diff
@@ -49,59 +49,41 @@ if { ! $free_insts_flag } {
 	exit 0
 }
 
-gpl::set_verbose_level_cmd 1
+read_lib $::env(LIB_SYNTH)
 
-gpl::set_density_cmd $::env(PL_TARGET_DENSITY)
+set arg_list [list]
 
-read_lib $::env(LIB_SYNTH)
+lappend arg_list -verbose_level 1
+lappend arg_list -density $::env(PL_TARGET_DENSITY)
 
 if { $::env(PL_BASIC_PLACEMENT) } {
-	gpl::set_overflow_cmd 0.9
-	gpl::set_init_density_penalty_factor_cmd 0.0001
-	gpl::set_initial_place_max_iter_cmd 20
-	gpl::set_bin_grid_cnt_x_cmd 64
-	gpl::set_bin_grid_cnt_y_cmd 64
+	lappend arg_list -overflow 0.9
+	lappend arg_list -init_density_penalty 0.0001
+	lappend arg_list -initial_place_max_iter 20
+	lappend arg_list -bin_grid_count 64
 }
 
 if { $::env(PL_TIME_DRIVEN) } {
 	read_sdc $::env(CURRENT_SDC)
-	gpl::set_timing_driven_mode 1
 	read_verilog $::env(yosys_result_file_tag).v
+	lappend arg_list -timing_driven
 } else {
-	gpl::set_timing_driven_mode 0
+	lappend arg_list -disable_timing_driven
```

Comment:
This will generate a "-disable_timing_driven is deprecated." warning.  You can just drop it.

Reply:
Yes, I noticed. I'm removing them.